sockeye/transformer.py

```diff
@@ -120,29 +122,41 @@ def hybrid_forward(self, F, data: mx.sym.Symbol, bias: mx.sym.Symbol) -> mx.sym.
 
 class TransformerDecoderBlock(mx.gluon.HybridBlock):
     """
-    A transformer encoder block consists self-attention, encoder attention, and a feed-forward layer
+    A transformer decoder block consists of an autoregressive attention block, encoder attention, and a feed-forward layer
     with pre/post process blocks in between.
     """
 
     def __init__(self,
                  config: TransformerConfig,
+                 inference_only: bool,
                  prefix: str,
                  dtype: str) -> None:
         super().__init__(prefix=prefix)
+        self.decoder_type = config.decoder_type
```

Comment:
do we need that attribute? Accessing `config` below just like for the other arguments seems fine to me.

Reply:
Definitely, I think it's a leftover from some previous iteration. Good catch!